src/shared/shared-kernel/src/fh-shared-kernel.shared-kernel/OpenReferral/Entities/Schedule.cs

```diff
@@ -1,87 +1,59 @@
-using System.ComponentModel.DataAnnotations.Schema;
 using System.Text.Json.Serialization;
 
 namespace FamilyHubs.SharedKernel.OpenReferral.Entities;
 
-public class Schedule
+public class Schedule : BaseHsdsEntity
 {
-    [JsonPropertyName("id")]
-    public required Guid Id { get; init; }
-
-    [JsonPropertyName("service_id")]
     [JsonIgnore]
     public Guid? ServiceId { get; init; }
 
-    [JsonPropertyName("location_id")]
     [JsonIgnore]
     public Guid? LocationId { get; init; }
 
-    [JsonPropertyName("service_at_location_id")]
     [JsonIgnore]
     public Guid? ServiceAtLocationId { get; init; }
 
-    [JsonPropertyName("valid_from")]
-    public DateTime? ValidFrom { get; init; }
+    [JsonPropertyName("valid_from")] public DateTime? ValidFrom { get; init; }
 
-    [JsonPropertyName("valid_to")]
-    public DateTime? ValidTo { get; init; }
+    [JsonPropertyName("valid_to")] public DateTime? ValidTo { get; init; }
 
-    [JsonPropertyName("dtstart")]
-    public DateTime? Dtstart { get; init; }
+    [JsonPropertyName("dtstart")] public DateTime? Dtstart { get; init; }
 
-    [JsonPropertyName("timezone")]
-    public byte? Timezone { get; init; }
+    [JsonPropertyName("timezone")] public byte? Timezone { get; init; }
 
-    [JsonPropertyName("until")]
-    public DateTime? Until { get; init; }
+    [JsonPropertyName("until")] public DateTime? Until { get; init; }
 
-    [JsonPropertyName("count")]
-    public short? Count { get; init; }
+    [JsonPropertyName("count")] public short? Count { get; init; }
 
-    [JsonPropertyName("wkst")]
-    public string? Wkst { get; init; }
+    [JsonPropertyName("wkst")] public string? Wkst { get; init; }
 
-    [JsonPropertyName("freq")]
-    public string? Freq { get; init; }
+    [JsonPropertyName("freq")] public string? Freq { get; init; }
 
-    [JsonPropertyName("interval")]
-    public short? Interval { get; init; }
+    [JsonPropertyName("interval")] public short? Interval { get; init; }
 
-    [JsonPropertyName("byday")]
-    public string? Byday { get; init; }
+    [JsonPropertyName("byday")] public string? Byday { get; init; }
 
-    [JsonPropertyName("byweekno")]
-    public string? Byweekno { get; init; }
+    [JsonPropertyName("byweekno")] public string? Byweekno { get; init; }
```

Comment:
Should that be ByWeekNo for pascal format? Same for others below

Reply:
Resolved in: [732d3da](https://github.com/DFE-Digital/fh-services/pull/111/commits/732d3da30f45cb57f65a875bc2f6713b690e07ee)